go/vt/vttablet/tabletserver/vstreamer/testenv/testenv.go

```diff
@@ -182,11 +182,11 @@ func (te *Env) RemoveAnyDeprecatedDisplayWidths(orig string) string {
 	}
 	var adjusted string
 	baseIntType := "int"
-	intRE := regexp.MustCompile(`(i)?int\(([0-9]*)?\)`)
+	intRE := regexp.MustCompile(`(?i)int\(([0-9]*)?\)`)
```

Comment:
if this definition comes from `information_schema` then it's guaranteed to be lower case.

Reply:
Not sure that it's guaranteed, but AFAIK it would be today. Have it in there just to be sure.

Reply:
:+1: